packages/devtools_app/lib/src/screens/inspector_v2/inspector_tree_controller.dart

```diff
@@ -1128,37 +1132,55 @@ class _RowPainter extends CustomPainter {
 
   @override
   void paint(Canvas canvas, Size size) {
-    double currentX = 0;
     final paint = _defaultPaint(colorScheme);
 
     final InspectorTreeNode node = row.node;
     final bool showExpandCollapse = node.showExpandCollapse;
+    final distanceFromExpandCollapseToRowStart = inspectorColumnIndent * 0.68;
```

Comment:
Can you put 0.68 in a named const?

Reply:
Done, did the same for 0.15 below as well